AzureCalling/Controllers/CallViewController.swift

```diff
@@ -578,13 +585,30 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
         infoHeaderView.updateParticipant(count: callingContext.participantCount)
     }
 
+    private func participantListUpdate() {
+        DispatchQueue.main.async { [weak self] in
+            guard let self = self,
+                  let bottomDrawerViewController = self.bottomDrawerViewController,
+                  bottomDrawerViewController.isViewLoaded else {
+                return
+            }
+
+            let participantListDataSource = ParticipantListDataSource()
+            let participantInfoList = self.getParticipantInfoList()
+            participantListDataSource.createParticipantList(participantInfoList)
+            bottomDrawerViewController.refreshBottomDrawer(dataSource: participantListDataSource)
+        }
+    }
+
     @objc func onRemoteParticipantsUpdated(_ notification: Notification) {
         queueParticipantViewsUpdate()
+        participantListUpdate()
```

Comment:
Have you considered whether or not it made sense to put this method into `queueParticipantViewsUpdate`?

Reply:
I thought about it before, but decided to separate them into their respective methods since they both handle different parts of the updates: one is to update specifically the active participants view, another to just update the list. Separation of concerns in a way.

There's also the delayed update mechanism for `updateParticipantViews`, unless we want the delay to happen when updating the participant list too, in which case combining both updates might make sense.